### PR TITLE
sys-cluster/zookeeper-bin: bump to 3.7.0 and fix GLEP 81

### PR DESCRIPTION
**Pull request by scardracs**
Closes: https://bugs.gentoo.org/678310
Closes: https://bugs.gentoo.org/781503

Package-Manager: Portage-3.0.19, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

**Comment by scardracs** [comment]
Please remember to change IDs on merge

File: acct-group/zookeeper/zookeeper-0.ebuild [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-group
+
+DESCRIPTION="Group for sys-cluster/zookeeper-bin"
+
+ACCT_GROUP_ID=217
```
Comment (ConiKost) [inline acct-group/zookeeper/zookeeper-0.ebuild:10]: This uid is already taken. Please will a PR at https://github.com/gentoo/api-gentoo-org/tree/master/files. Use `used_free_uidgids.sh` to get a free uid/gid combo.

File: acct-user/zookeeper/zookeeper-0.ebuild [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for sys-cluster/zookeeper-bin"
+
+ACCT_USER_ID=217
+ACCT_USER_GROUPS=( zookeeper )
```
Comment (ConiKost) [inline acct-user/zookeeper/zookeeper-0.ebuild:11]: This uid is already taken. Please will a PR at https://github.com/gentoo/api-gentoo-org/tree/master/files. Use `used_free_uidgids.sh` to get a free uid/gid combo.

File: sys-cluster/zookeeper-bin/zookeeper-bin-3.7.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror binchecks"
+
+RDEPEND="
+	acct-group/zookeeper
+	acct-user/zookeeper
+	>=virtual/jre-1.8
+"
+
+S="${WORKDIR}/apache-${MY_P}-bin"
```
Comment (ConiKost) [inline sys-cluster/zookeeper-bin/zookeeper-bin-3.7.0.ebuild:26]: S after SRC_URI.

**Review by ConiKost — CHANGES_REQUESTED** [review]

File: acct-group/zookeeper/zookeeper-0.ebuild (hunk 1 above)
Comment (ionenwks) [reply acct-group/zookeeper/zookeeper-0.ebuild:10]: Lately I think proxy-maint just been picking & updating the uid/gid on merge, PR process delays notably make it harder to keep this in sync too.

Comment (scardracs) [reply acct-group/zookeeper/zookeeper-0.ebuild:10]: Yeah it is, I've opened these PR ca 1 month ago

Comment (ConiKost) [reply acct-group/zookeeper/zookeeper-0.ebuild:10]: Not a problem, I will look later today here and will just reserve some uid/gid.

**Pull request closed without merging**